front/app/containers/IdeasNewPage/WithJSONForm/Heading.tsx

```diff
@@ -130,8 +130,9 @@ export const Heading = ({
             width="100%"
             color={headingTextColor}
             variant="bodyL"
-            fontWeight="normal"
-            fontSize={isSurveyOnMobile ? 'xl' : 'xxxxl'}
+            fontWeight={isSurveyOnMobile ? 'bold' : 'normal'}
+            fontSize={isSurveyOnMobile ? 'l' : 'xxxxl'}
+            ml={isSurveyOnMobile ? '16px' : '0px'}
```

Comment:
Centers the "Survey" title and uses a smaller + bold font to match Ben's design.